### ISSUE TITLE
Detect flight tickets purchased for non-authorized people

### ISSUE THREAD

**Issue by Irio** [datasciencebr/serenata-de-amor#138]
The [Bureau Act](https://github.com/datasciencebr/serenata-de-amor/blob/83971d9d453ef1ff248b1b5a6b466754fe13995a/CEAP.md) regulating the CEAP allows purchases for the congresspeople and their advisors. When it's not the case, the reimbursement should be denied.

**Comment by xlucasdemelo** [comment]
What is your point with this issue? Sorry for my bad english

**Comment by cuducos** [comment]
@xlucasdemelo if we have a list of the official advisors (that's what #139 is about) we can check in each flight ticket if the name of the passenger (field `passenger`) is either:

1. the congressperson herself/himsels
2. or one of the the official advisors

If the `passenger` is someone different it is probably a illegal expense.

**Comment by fgrehm** [comment]
Played with this data yesterday and noticed one thing, the `passenger` field is usually set for tickets bought for other people, going to share some basic analysis I did on it at some point this week

**Comment by fgrehm** [comment]
Here's a pretty simple notebook that outlines the data we have around flight tickets and passengers: https://gist.github.com/fgrehm/5fc51a3c0be3799dded3ae9f93f04b99

#### TL;DR

- Looks like passenger is _usually_ set for tickets bought for other people
- Out of 977 reimbursements that have the passenger field set on 2015-2016, only 14 can be matched to the congressperson civil name from the deputies dataset, the rest should be tickets bought for other people, summing up R$ 824.118,18

**Comment by zmechz** [comment]
@fgrehm Usually, when you buy a flight ticket, you have a billing data and the flight ticket data, meaning billing name is not necessarily the same as the passenger's name.
Do you guys have both data (billing + fligh ticket) in CSVs?

**Comment by cuducos** [comment]
> Do you guys have both data (billing + fligh ticket) in CSVs?

The CSV offers only the passenger name. The receipt image might have more info, so the dataset generated with OCR data might be useful too.

**Comment by arphoton** [comment]
Hi guys,
I'm new here and I'm very interested in this issue.
Please, somebody could send me an update about it? 

Thanks!

**Comment by cuducos** [comment]
> Please, somebody could send me an update about it?

Hi @arphoton — welcome : )

I'm not sure about what you mean by an _update_. We keep the discussion of each issue in its page here on GitHub, so I'm afraid all we have is in this very page.

Surely as you're new you can check the `CONTRIBUTING.md` to check auxiliary datasets that might help in this issue, and join the Telegram group (link in the `CONTRIBUTING.md`) if you wanna yet another channel to interact with developers around here ; )
